src/client/mod.rs

```diff
@@ -140,7 +140,8 @@ impl Reader for Context {
                         // NOTE: A call to `Client::call` implementation *MUST* always return the `Response` variant matching the `Request` one.
                         // TIPS: This can be ensured via a call to `verify_response_header`( in 'src/service/mod.rs') before returning from `Client::call`.
                         unreachable!(
-                            "unexpected response code: {} (request code: {})",
+                            "unexpected response code: {} (request code: {})\n\
+                            note: Please fill an issue at `https://github.com/slowtec/tokio-modbus/issues` with a minimal example reproducing this bug.",
```

Comment:
"fill" -> "file"?

This verbose message is duplicated many times. Could this be avoided by externalizing the composition of the error message into a function? Does not need to be efficient, because it should never be called.